__tests__/app.test.js

```diff
@@ -176,3 +178,42 @@ describe("post/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("Patch/api/articles/:article_id", () => {
+  test("PATCH:200 update article and respond with updated article incremented by 1", () => {
+    const newVote = { inc_votes: -50 };
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVote)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body[0]).hasOwnProperty("article_img_url");
+        expect(body[0]).hasOwnProperty("author");
+        expect(body[0]).hasOwnProperty("body");
+        expect(body[0]).hasOwnProperty("created_at");
+        expect(body[0]).hasOwnProperty("title");
+        expect(body[0]).hasOwnProperty("topic");
+        expect(body[0].votes).toBe(50);
+      });
+  });
+  test("POST:404 should respond with correct msg for un-found id", () => {
+    const newVote = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/999")
+      .send(newVote)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
+  });
+  test("POST:400 should respond with correct msg if no body is given", () => {
+    return request(app)
+      .patch("/api/articles/999")
+      .send()
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
+  });
```

Comment:
For your 400 test, you have done both 'incorrect' things - i.e. not sent a body and an unknown id. It's best practice to only do one thing at a time :) 

400 - invalid id